docs/tutorials/index.md

```diff
@@ -1,7 +1,16 @@
 # Tutorials
 
-This library is under active development and tutorials are not yet available.
+## Learn the Basics
 
-In the meantime, you can check out some [basic examples](../getting_started/examples.md), or the [Python API](../python_api/index.md) for documentation on individual functions featured in dynamiqs.
+Familiarize yourself with dynamiqs. Learn how to run simulations of quantum differential equations, define time-dependent objects, batch over many parameters, or compute gradients.
```

Comment:
I thing the phrasing "quantum differential equations" is not very standard, here's a proposed reformulation 😊
```suggestion
Familiarize yourself with dynamiqs. Learn how to simulate quantum systems, define time-dependent operators, run multiple simulations simultaneously, or compute gradients.
```